src/satellite/config.ts

```diff
@@ -91,10 +91,24 @@ export const validateConfig = (config : any) => {
     errors.push("Please provide config.replication = {address, port} details to connect to dev infra")
     errors.push("We're still working to make ElectricSQL service live. You can join the wait list: https://console.electric-sql.com/join/waitlist")
   } else{
-    const {address, port } = replication
-    if(!address) {errors.push(`Please provide config.replication.address`)}
-    if(!port) {errors.push(`Please provide config.replication.port`)}
-    if(port && typeof port != 'number') {errors.push(`Please provide correct type for config.replication.port`)}
+    const { address, port, insecure } = replication
+
+    if (!address) {
+      errors.push(`Please provide config.replication.address`)
+    }
+    if (!port) {
+      errors.push(`Please provide config.replication.port`)
+    }
+    if (port && typeof port != 'number') {
+      errors.push(`Please provide correct type for config.replication.port`)
+    }
+    if (insecure && env != 'dev') {
```

Comment:
This was not supposed to go on this commit because insecure flag is only added in VAX-449

Reply:
- [ ] Need to revert this change because environment names are not fixed